### PR TITLE
Add tests for colony actions (REMIX)

### LINKED ISSUE #3250: Add a set of tests for actions - part 2
Splitting up the initial issue #3205 as otherwise it would be too much.

Actions that need to be done in this issue:

- unlock token
- manage permissions
- recovery
- upgrade

### PR DESCRIPTION
**Pull request by ArmandoGraterol**
Added tests for the following actions:

- Unlock native token
- Permission management
- Recovery mode

For the permission management, make sure to have at least 2 members in the colony.

Resolves #3250

### CODE REVIEW

**Review by chinins — COMMENTED** [review]
Bice work! A couple of points:
- comment on the unlock token had an error
Screenshot 2022-03-25 at 15 51 27

- the heading  in the permission management selector didn't work
Screenshot 2022-03-25 at 15 56 29

**Comment by chinins** [comment]
Oh and same question about the camel case.

File: cypress/e2e/3-actions.js [hunk 1]
```diff
@@ -341,4 +341,121 @@ describe('User can create actions via UAC', () => {
...
     });
   });
+  it('Can unlock the native token', () => {
+    const annotationText = 'Time to unlock the token';
+
+    cy.login();
+
+    cy.visit(`/colony/${Cypress.config().colony.name}`);
+
+    cy.getBySel('new-action-button', { timeout: 60000 }).click();
```
[the remark this answers is not in the document]
Comment (ArmandoGraterol) [reply cypress/e2e/3-actions.js:351]: Indeed! Forgot to update them

**Comment by ArmandoGraterol** [comment]
> * comment on the unlock token had an error

@chinins  Yeah, that is because of this: #3085 

I added the annotation check anyways because it is after all a test to check that everything is working as expected. Do you think that it would be better to remove it if the annotation is not working, for now at least?

@arrenv might want to pitch into this as well

**Comment by chinins** [comment]
> > * comment on the unlock token had an error
> 
> @chinins Yeah, that is because of this: #3085
> 
> I added the annotation check anyways because it is after all a test to check that everything is working as expected. Do you think that it would be better to remove it if the annotation is not working, for now at least?
> 
> @arrenv might want to pitch into this as well

Yeah, now I remember this issue. let's just remove it, at least for now. We test annotations everywhere else so it's not a big deal at all.

Once we decide on an approach to dialog menu selectors I'll and you remove the annotation for the unlock token, i'll retest again and approve.

**Review by chinins — APPROVED** [review]
Looking good - very nice work!

**Pull request merged**